packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -4102,6 +4200,7 @@ abstract class Element extends DiagnosticableTree implements BuildContext {
       // (the root node), the owner should have already been assigned.
       // See RootRenderObjectElement.assignOwner().
       _owner = parent.owner;
+      _buildScope = parent.buildScope;
```

Comment:
Yeah. `_buildScope` is the `parent.buildScope` cache, and should not be read by anyone except `buildScope`.

Reply:
Makes sense, as suggested in another comment, maybe we should rename _buildScope to avoid that it is being confused as the backing store for the buildScope getter, which it only sometimes is.